e2e/utils.go

```diff
@@ -207,9 +212,15 @@ func getMountType(appName, appNamespace, mountPath string, f *framework.Framewor
 //  * issue get request for particular key
 // resulting in stdOut (first entry in tuple) - output that contains the key
 // or stdErr (second entry in tuple) - error getting the key.
-func readVaultSecret(key string, f *framework.Framework) (string, string) {
+func readVaultSecret(key string, usePassphrasePath bool, f *framework.Framework) (string, string) {
```

Comment:
are we setting usePassphrasePath to true or false anywhere? not able to find it

Reply:
See lines 261 + 274 in e2e/rbd_helper.go, depending on the KMS-ID (`vault`/`vaulttokens`) `usePassphrasePath` is set to `true` or `false`.